translations/en.json

```diff
@@ -258,6 +294,11 @@
         "website": "Website",
         "website_plural": "Websites"
     },
+    "member_displayer_list_item_component": {
+        "snackbar": {
+            "email_copied_to_clipboard": "Email copied to clipboard."
+        }
+    },
```

Comment:
This string has been replaced by the snackbar.email_copy_to_clipboard one, please remove it